### PR TITLE
Change telemed report frequency

### PR DESCRIPTION
**Pull request by prabhanshuguptagit**
**Story card:** -

## Because

We want to send the telemed report every month instead of every week.

## This addresses

Changes the frequency to once a month.

### CODE REVIEW

File: doc/wiki/feature-flags.md [hunk 1]
```diff
@@ -24,5 +24,5 @@ to facility the launch of a new feature, and may reference the Simple team's int
 | organization_reports | Yes | When enabled, the Simple Dashboard will allow users to view a top-level report for the whole organization. |
 | skip_api_validation | Yes | When enabled, Simple Server will not perform JSON validation of incoming API requests. This makes the API endpoints much more performant, but should only be enabled if all API clients are trusted (Simple app only). |
 | sync_encounters | Yes | When enabled, encounters can be synced between app and server like any other standard sync resource. |
-| weekly_telemed_report | Yes | When enabled, a weekly report on telemedicine activity in Simple will be sent via email to the configured recipients. |
+| automated_telemed_report | Yes | When enabled, a monthly report on telemedicine activity in Simple will be sent via email to the configured recipients. |
```
Comment (harimohanraj89) [inline doc/wiki/feature-flags.md:27]: `monthly_telemed_report` or `automated_telemed_report`?

**Review by harimohanraj89 — APPROVED** [review]
One doc change - but preemptive approval

File: doc/wiki/feature-flags.md (hunk 1 above)
Comment (prabhanshuguptagit) [reply doc/wiki/feature-flags.md:27]: Called it `automated_` because we'll have to change name again if the frequency changes.

Comment (harimohanraj89) [reply doc/wiki/feature-flags.md:27]: Right, but the feature flag in the scheduler says `monthly_` :smile:

Comment (prabhanshuguptagit) [reply doc/wiki/feature-flags.md:27]: Lol. Wow. Thanks.

**Pull request merged**